lib/spack/spack/test/mirror.py

```diff
@@ -149,6 +151,100 @@ def test_all_mirror(
     repos.clear()
 
 
+@pytest.mark.parametrize(
+    "mirror",
+    [
+        spack.mirror.Mirror(
+            'https://example.com/fetch',
+            'https://example.com/push',
+        ),
+    ],
+)
+def test_serde_mirror(mirror):
```

Comment:
I think this should be `serdes`, but I could definitely be wrong. I think `serde` is the Rust framework for `serdes`.

In other test modules, the serdes methods are named `*_roundtrip_[lang]` so we haven't had to think about it before.

Reply:
Yes, `serde` is the rust framework for de/serialization based on the colloquial term! I have removed this testing as it seems unrelated to the thrust of the PR.